ADAL/tests/unit/ios/ADKeychainTokenCacheTests.m

```diff
@@ -403,11 +403,11 @@ - (void)testRemoveAllForUserId
     XCTAssertEqual([self tombstoneCount], 0);
     
     //remove items with user ID as @"eric@contoso.com" and client ID as TEST_CLIENT_ID
-    [mStore removeAllForUserId:@"eric@contoso.com" error:&error];
+    [mStore wipeAllItemsForUserId:@"eric@contoso.com" error:&error];
     ADAssertNoError;
     XCTAssertEqual([self count], 2);
-    XCTAssertEqual([self tombstoneCount], 2);
-    //only item3 and item4 are left in cache while the other twi should be tombstones
+    XCTAssertEqual([self tombstoneCount], 0);
```

Comment:
nit: would be nice to also have a test that would check deleting pre-existing tombstones

Reply:
But we will get rid of the tombstone altogether